index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Landing Page</title>
+  </head>
+  <body>
+    <div class="header">
+      <div class="wrapper">
+        <!-- Navbar Start -->
+
+        <div class="navbar">
+          <a class="logo-img" href="#"
+            ><img src="images/logo.png" alt="logo"
+          /></a>
+          <ul class="nav-links">
+            <li><a href="#">Home</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#">About us</a></li>
+            <li><a href="#">Contact us</a></li>
+          </ul>
+        </div>
+
+        <!-- Navbar End -->
+
+        <!-- Header main section Start -->
+
+        <div class="header-main">
```

Comment:
I would like to explain that instead of using **div** you can use **HTML 5** tags (header, section, form) as its a good and latest approach.

Reply:
Please make sure that the comments are clear and easy to understand like in your comment you mentioned "its a good and latest approach" but why is it good. Why are we using this latest approach? it doesn't explain . Here is an example for you to understand the difference between clear and vague comment:

* This is a clear comment: *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*
* This is a vague comment: *The README is not descriptive, why don't you make it better?*

Here is a way to improve your comment:

"Thank you for your work on this project. I noticed that you used div elements for the layout of the page. I suggest that you use HTML 5 semantic tags (header, section, form) instead. This will make your code more readable, accessible, and SEO-friendly. You can learn more about HTML 5 semantic tags [here](https://www.w3schools.com/html/html5_semantic_elements.asp)"

Reply:
- [Optional] Great job for updating based on the previous code reviewer 👍 , I think it would be better if you use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.